pkg/rke/k8s_rke_system_images.go

```diff
@@ -10226,6 +10226,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled out of band post v2.8.4
```

Comment:
April patches are getting released with v2.8.4 so this would be // Enabled in Rancher v2.8.4

Reply:
updated @kinarashah